src/SFA.DAS.AssessorService.Web/Validators/Standard/OptOutStandardVersionViewModelValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Linq;
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Http;
+using SFA.DAS.AssessorService.Application.Api.Client.Clients;
+using SFA.DAS.AssessorService.Web.Helpers;
+using SFA.DAS.AssessorService.Web.ViewModels.Standard;
+
+namespace SFA.DAS.AssessorService.Web.Validators.Standard
+{
+    public class OptOutStandardVersionViewModelValidator : AbstractValidator<OptOutStandardVersionViewModel>
```

Comment:
This should include the code in the existing view model validator - there is no need for 2 on the same view model.